Sources/PubNub/Errors/ErrorDescription.swift

```diff
@@ -197,7 +197,7 @@ extension PubNubError.Reason: CustomStringConvertible, LocalizedError {
     case .unauthorized:
       return "Access was denied due to insufficient authentication/authorization"
     case .forbidden:
-      return "Authorization key is missing or does not have the permissions required to perform this operation"
+      return "No permissions required to perform this operation"
```

Comment:
This doesn't sound right. It literally says that there is no special permissions required, when we get at 403.

Reply:
I will change it to: `"Insufficient permissions to perform this operation`"